amazon-sqs/5.8/modules/ROOT/pages/amazon-sqs-connector-reference.adoc

```diff
@@ -282,33 +336,36 @@ Deletes the message identified by message object on the queue this object repres
 |Type |String
```

Comment:
Same applies here, String returned by this operation contains ID of the request, shouldn't we mention it here?